exercises/practice/yacht/.meta/example.rb

```diff
@@ -0,0 +1,77 @@
+class Yacht
+  attr_reader :dice, :category
+
+  def initialize(dice, category)
+    @dice = dice.sort
+    @category = category
+  end
+
+  def score
+    case category
+    when 'ones' then count_occurances(1)
+    when 'twos' then count_occurances(2) * 2
+    when 'threes' then count_occurances(3) * 3
+    when 'fours' then count_occurances(4) * 4
+    when 'fives' then count_occurances(5)  * 5
+    when 'sixes' then count_occurances(6) * 6
+    when 'yacht' then yacht
+    when 'full house' then full_house
+    when 'four of a kind' then four_of_a_kind
+    when 'little straight' then little_straight
+    when 'big straight' then big_straight
+    else choice
+    end
+  end
+
+  private
+
+  def count_occurances(num)
+    return dice.tally({})[num] || 0
```

Comment:
```suggestion
    dice.tally({})[num] || 0
```

Also, prefer `number` rather than `num` for reasons of abbreviations.